lib/hooks/userspace/hook-express.js

```diff
@@ -70,14 +72,12 @@ function middleware(req, res, next) {
 /**
  * Creates and sets up a new root span for the given request.
  * @param {Object} req The request being processed.
+ * @param {Object} traceHeader The incoming trace header.
```

Comment:
traceHeader should always be defined at this point. See https://github.com/GoogleCloudPlatform/cloud-trace-nodejs/pull/217/files#diff-aa20dec8ea838bb18b5bb09bd5b9fce8R45